portfolio/src/main/java/com/google/sps/servlets/CommentBlobstoreServlet.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comment-blobstore")
+public class CommentBlobstoreServlet extends HttpServlet {
+
+  // Use constant redirect instead of client-supplied redirect URL 
+  // to avoid blobstore redirect spoofing.
```

Comment:
Is there a link to learn more about this?

Reply:
Hm. I'm thinking this comment sounds more official than it actually is. I was just reasoning that there are less security concerns if the redirect URL is hard-coded vs. client-supplied.

Now that you bring it up, I'm wondering if this comment is really even relevant. It was more of an overall design decision versus an implementation decision.